charts/kube-prometheus-stack/charts/grafana/values.yaml

```diff
@@ -630,12 +630,12 @@ sidecar:
     # If specified, the sidecar will search for dashboard config-maps inside this namespace.
```

Comment:
Why do you edit the grafana value file and not the `charts/kube-prometheus-stack.yaml` helm option file directly ?

Reply:
Heh, good point, indeed one isn't supposed to make changes to the vendored files unless really necessary.

Reply:
Oh. That's just my weak knowledge of Helm. I saw `Values`, so I edited `Values` :sweat_smile: